### DISCUSSION TITLE
Laravel Multipayment Gateways

### DISCUSSION THREAD

**Discussion by MusahMusah** [HassanZahirnia/laravel-package-ocean#47] [Package Suggestions]
https://github.com/MusahMusah/laravel-multipayment-gateways

A Laravel Package that makes implementation of multiple payment Gateways endpoints.
The package currently supports multiple gateways such as Paystack, Flutterwave and Stripe. The package offers an easy to use interface that abstracts the complexities of integrating with these payment gateways. It also provides a way to handle webhooks from the payment gateways.

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊, I've added the package! It will be live in a moment.
